README.md

```diff
@@ -61,7 +61,7 @@ which normatively accepts SPIR-V but does not normatively consume a high-level s
 - [GL_NV_shader_sm_builtins](https://github.com/KhronosGroup/GLSL/blob/master/extensions/nv/GLSL_NV_shader_sm_builtins.txt)
 - [GL_EXT_demote_to_helper_invocation](https://github.com/KhronosGroup/GLSL/blob/master/extensions/ext/GLSL_EXT_demote_to_helper_invocation.txt)
 - [GL_EXT_shader_realtime_clock](https://github.com/KhronosGroup/GLSL/blob/master/extensions/ext/GL_EXT_shader_realtime_clock.txt)
-- [GLSL_EXT_buffer_reference_uvec2](https://github.com/KhronosGroup/GLSL/blob/master/extensions/ext/GLSL_EXT_buffer_reference_uvec2.txt)
+- [GL_EXT_buffer_reference_uvec2](https://github.com/KhronosGroup/GLSL/blob/master/extensions/ext/GLSL_EXT_buffer_reference_uvec2.txt)
```

Comment:
I'd rather we keep unrelated fixes in a separate MR to reduce the possibility of confusion in the future. (These two renamings do seem correct, though).

Reply:
ok - I split that out to https://github.com/dgkoch/GLSL/pull/1 which I'll retarget after this one is merged.